src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -85,6 +86,16 @@ public bool IsAbleToGoHome
             get { return !(model.CurrentWorkspace is HomeWorkspaceModel); }
         }
 
+        public bool IncanvasSearchIsOpen
```

Comment:
I know, that this property must be in WorkspaceViewModel, but look at explanation down....